app/models/clickhouse/events_enriched.rb

```diff
@@ -10,15 +10,15 @@ class EventsEnriched < BaseRecord
 #
 # Table name: events_enriched
 #
-#  code                       :string           not null, primary key
```

Comment:
Not sure these change should be part of this PR (not even sure about these changes :D )

Reply:
Oh no. I assumed someone forgot to commit them, but I'll remove them. 
Locally, I'll delete my clickhouse DB and remigrate.